packages/netlify-cms-backend-bitbucket/src/API.ts

```diff
@@ -448,13 +449,18 @@ export default class API {
   }
 
   async getDifferences(branch: string) {
-    const diff: BitBucketDiffStat = await this.requestJSON({
-      url: `${this.repoURL}/diffstat/${branch}..${this.branch}`,
+    const rawDiff = await this.requestText({
+      url: `${this.repoURL}/diff/${branch}..${this.branch}`,
       params: {
-        pagelen: 100,
+        binary: false,
       },
     });
-    return diff.values;
+
+    return parse(rawDiff).map(d => ({
+      newPath: d.newPath.replace(/b\//, ''),
+      binary: d.binary,
```

Comment:
Are `svg` images consider as binary?

Reply:
Whoops, looks like they are not considered as binary.